README.md

```diff
@@ -1,16 +1,32 @@
 [![Build Status](https://travis-ci.com/mdellweg/ansible_modules_pulp.svg?branch=master)](https://travis-ci.com/mdellweg/ansible_modules_pulp)
 
-ansible modules for Pulp aka Squeeze
+ansible modules for Pulp aka Squeezer
 ===
 
 This project project aims to provide a complete set of ansible modules to control a pulp3 server in a descriptive way.
 This is neither to be confused with ansible-pulp to install pulp, nor pulp\_ansible to manage ansible content in pulp.
+It has been wildly inspired by the sister project [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules).
 
 Testing
 ---
 
-Testing infrastructure has been wildly inspired by the sister project [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules).
-Look at the documentation there for now.
+Testing is done by running handcrafted playbooks while playing back prerecorded server answers.
+
+Those playbooks are meant to test one specific module, and are usually structured in three sections:
```

Comment:
"The playbooks under `tests/playbooks`"?

Reply:
good catch!